### ISSUE TITLE
Special Characters in Filenames Logic Issue

### ISSUE THREAD

**Issue by avirosso** [natefrisch01/Graph-Link-Types#7]
First: Great plugin idea!

One problem: In Obsidian, 'Inline-Links' could be in wikilink-format or in markdown-link-format. Unfortunately, this plugin don't work with markdown-links:

Works:
`Author:: [[Ludwig Wittgenstein]]`

Don't work:
`Author:: [Ludwig Wittgenstein](Ludwig%20Wittgenstein.md)`

Please support markdown-links (obsidian is a markdown editor and many users prefer markdown-links to be standard compliant)

Thanks in advance!

**Comment by natefrisch01** [comment]
Thanks for pointing this out! It should be a quick fix, give me a few minutes here...

**Comment by natefrisch01** [comment]
Should be fixed! If it works for you, let me know and I'll close the issue.

**Comment by avirosso** [comment]
Thanks for this quick fix!

But unfortunately it doesn't work (it only works if I link a note in vault root and the note name has no spaces or special characters ....)

Possible reasons
- In obsidian link setting I have: "Shortest path when possible" (will use only the note name, unless there's two notes with the same names).
- If the note name contains spaces or special characters, they will be urlencoded in markdown-links)

Hope it helps!

Thanks in advance!

**Comment by natefrisch01** [comment]
Lol, sorry, let me try again

**Comment by natefrisch01** [comment]
Alright, so, it should work for files inside of folders now, but as for special characters, I'll need to look more closely at what obsidian allows. I'll work on it soon - thanks for the help!

**Comment by avirosso** [comment]
Hi, 

what are the chances that this problem will be solved in the near future? (unfortunately this  cool plugin is currently not usable for me ...)

Thanks in advance

**Comment by natefrisch01** [comment]
I'm planning a rewrite of the entire code base for handling all the link formats, since most of the bugs are with handling those or special characters, files with the same name being in different folders, etc. it should be with the next release, but I'm not sure how long it will take since there are a lot of cases to handle.
